iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/helplets.inl

```diff
@@ -128,18 +129,18 @@ inline bool isValidPathToDirectory(const string<StringCapacity>& name) noexcept
         // with a valid file name
         if (separatorPosition)
         {
-            auto filenameToVerify = temp.substr(0, *separatorPosition);
-            bool isValidDirectory = isValidFileName(*filenameToVerify) || *filenameToVerify == currentDirectory
-                                    || *filenameToVerify == parentDirectory;
+            auto filenameToVerify{temp.substr(0, *separatorPosition)};
+            bool isValidDirectory{isValidFileName(*filenameToVerify) || *filenameToVerify == currentDirectory
+                                  || *filenameToVerify == parentDirectory};
```

Comment:
Axivion also complains about the missing parentheses and `const`. Please add them.

Reply:
@FerdinandSpitzschnueffler when I have added `const` additionally reported 2 more `AutosarC++19_03-A8-4-7` violations.

Reply:
For which parameters? If it's not easy to fix now, you can also remove the `const` again and we deal with this in another PR.